packages/object-lists/migrations/0000_bored_silver_surfer.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE `object_list_item` (
+	`id` serial AUTO_INCREMENT NOT NULL,
+	`object_id` varchar(256) NOT NULL,
+	`list_id` int,
```

Comment:
Is this a reference to the `object_list` table? If so, rename to `object_list_id` (to prevent confusion with future 'list' tables that we might get)?